index.d.ts

```diff
@@ -23,7 +23,7 @@ interface ConfigParams {
      * REQUIRED. The secret(s) used to derive an encryption key for the user identity in a session cookie.
      * Can use env key APP_SESSION_SECRET instead.
      */
-    appSessionSecret: boolean | string | string[];
+    appSessionSecret?: boolean | string | string[];
```

Comment:
Are there any tests needed around this?

Reply:
I don't know but probably not? Are there test you can write for a JS library TS definition file?

Reply:
Not so much the definition file but I meant the fact that the property is now optional. What happens when it's not supplied and and environment variable isn't set?

Reply:
Then you'll get a validation error on startup.